apps/andi/assets/css/app.scss

```diff
@@ -0,0 +1,49 @@
+/* This file is for your main application css. */
```

Comment:
```suggestion

```

Reply:
Resolved without change or comment??

Reply:
These comments are generated by Phoenix. We will be leaving them to help with onboarding the team to our new use of Phoenix server generated HTML.